qa-tests-backend/src/test/resources/logback-test.xml

```diff
@@ -2,7 +2,7 @@
     <withJansi>true</withJansi>
```

Comment:
Not sure what @rukletsov alluded to, but to me it looks like we can have a cake and eat it too.
See https://logback.qos.ch/manual/layouts.html#coloring

> Setting withJansi to true enables ANSI color code interpretation by the Jansi library, which transparently filters out ANSI escape sequences if the underlying terminal is not compatible. This is the safest choice for cross-platform deployments, **but requires org.fusesource.jansi:jansi:1.17** or higher on the class path.

We don't have a dependency on `org.fusesource.jansi:jansi`. Maybe that's all we need to enable color auto-sensing?

Reply:
See https://github.com/stackrox/stackrox/pull/3389